src/EnergyPlus/SZVAVModel.cc

```diff
@@ -0,0 +1,712 @@
+		//                |    |                   |    | ^               * = supply air flow rate
+		//                |    |                   |^^^^| <--- maximum supply air temperture
+		//                |    |                ^  |    |
+		//                |    |              ^    |    |
+		//     ***********|    |            ^      |    |**************   <-- max unit air flow rate
+		//                |*   |          ^        |   *|
+		//                | *  |        ^          |  * |
+		//                |  * |      ^            | *  |
+		//                |   *|    ^              |*   |
+		//                |    |*******************|    |                 <-- min unit air flow rate
+		//          R3    | R2 | ^       R1        | R2 |    R3
+		//   min SAT -->  |^^^^|                   |    |
+		//               ^
+		//             ^                  |
+		//  (-) increasing cooling load < 0 > increasing heating load (+)
+		//
+		// Notes: SAT is allowed to decrease/increase once air flow rate is max'ed out otherwise load would not be met
+		//        Load here is not the zone load, it's the load the system must meet to meet the Tstat set point (i.e., OA can alter required capacity)
+		//        lowSpeedFanRatio = min/max unit air flow rate
+		//
+		// Step 1: calculate load at Region 1 lower (cooling) or upper (heating) boundary at minimum air flow rate
+		//         - if load can be met, test maximum output (PLR = 1) before calling RootSolver
+		//         - if maximum capacity is greater than load, solve for PLR
+		// Step 2: calculate load at Region 3 lower (cooling) or upper (heating) boundary at maximum air flow rate
+		//         - if load is less than boundary load, solve for air flow and PLR that meet the load
+		//         - ELSE
+		// Step 3: solve for Region 3 PLR
+		//       DONE
+		//
+
+		// Step 1: Determine boundary for region 1
+		// calculate sensible load based on minimum air flow rate and specified supply air temperature limit
+		if ( SZVAVModel.ATMixerExists ) {
+			if ( SZVAVModel.ATMixerType == DataHVACGlobals::ATMixer_SupplySide ) {
+				// Air terminal supply side mixer
+				boundaryLoadMet = minAirMassFlow * ( PsyHFnTdbW( DataLoopNode::Node( SZVAVModel.ATMixerOutNode ).Temp, ZoneHumRat ) - PsyHFnTdbW( ZoneTemp, ZoneHumRat ) );
+			} else {
+				// Air terminal inlet side mixer
+				boundaryLoadMet = minAirMassFlow * ( PsyHFnTdbW( maxOutletTemp, ZoneHumRat ) - PsyHFnTdbW( ZoneTemp, ZoneHumRat ) );
+			}
+		} else {
+			minHumRat = min( DataLoopNode::Node( InletNode ).HumRat, DataLoopNode::Node( OutletNode ).HumRat );
+			boundaryLoadMet = minAirMassFlow * ( PsyHFnTdbW( maxOutletTemp, minHumRat ) - PsyHFnTdbW( InletTemp, minHumRat ) );
+		}
+
+		if ( ( CoolingLoad && boundaryLoadMet < ZoneLoad ) || ( HeatingLoad && boundaryLoadMet > ZoneLoad ) ) { // in Region 1 of figure
+																												// Step 1: set min air flow and full coil capacity
```

Comment:
What happened to this line?  I know your in Florida but no reason to make this comment so far East on my screen.  (Yeah yeah technically it's out west at my south facing desk)

Reply:
My editor right justifies comment lines. I don't know why and it pisses me off. Will fix.